monkey/common/cloud/environment_names.py

```diff
@@ -0,0 +1,7 @@
+UNKNOWN = "Unknown"
+ON_PREMISE = "On Premise"
+AZURE = "Azure"
+AWS = "AWS"
+GCP = "GCP"
+
+ALL_ENV_NAMES = [UNKNOWN, ON_PREMISE, AZURE, AWS, GCP]
```

Comment:
Can we figure out a way to dynamically generate this, or is this going to be so static theres no point?

Reply:
We can with `locals()` or something akin to that, but there's no point. 

I'll add some more clouds + comment for futureproofing

Reply:
Waste of time. I added most clouds + a comment instead.